### PR TITLE
chore: add conformance test coverage for typed function signature

### PR DESCRIPTION
**Pull request by garethgeorge**

### CODE REVIEW

**Comment by garethgeorge** [comment]
Conformance tests passing.

File: tests/conformance/main.py [hunk 1]
```diff
@@ -19,6 +19,20 @@ def from_dict(obj: dict) -> "ConformanceType":
         return ConformanceType(json.dumps(obj))
```
Comment (KaylaNguyen) [inline tests/conformance/main.py:19]: This seems no longer used.

**Review by KaylaNguyen — APPROVED** [review]

**Comment by kenneth-rosario** [comment]
nit(unrelated to your change): Can you update https://github.com/GoogleCloudPlatform/functions-framework-python/pull/254/files#diff-e5ed24bc2f5084a1664419e35473c15466f87e7552ef1fad4d13cdde0eaed400R2 to only do push when it is on branch master? It seems like it is being duplicated during pull requests. 
image

File: tests/conformance/main.py (hunk 1 above)
Comment (garethgeorge) [reply tests/conformance/main.py]: Good catch, deleted.

**Comment by garethgeorge** [comment]
@kenneth-rosario done, added a branch selector to the push trigger for our ci runs.

**Pull request merged**